client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,103 @@
-function RestaurantPreview({ restaurant }) {
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+
+const classes = {
+  previewContainer: {
+    height: "120px",
+    width: "100%",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "75%",
+    width: "100px",
+    margin: "1rem",
+    objectFit: "cover",
+  },
+  detailsContainer: {
+    marginLeft: "0.25rem",
+  },
+  nameContainer: {
+    fontSize: "1.5rem",
+    textDecoration: "none",
+    color: "black",
+  },
+  categoriesContainer: {
+    marginTop: "1rem",
+  },
+  reviewContainer: {
+    fontSize: "0.75rem",
+    display: "flex",
+  },
+  exitContainer: {
+    postion: "absolute",
+  },
+};
+
+function formatPhoneNumber(phoneNumberString) {
```

Comment:
Add a source if you found this online. Otherwise if you did this yourself - holy shit nice